### DISCUSSION TITLE
Track compressor

### DISCUSSION THREAD

**Discussion by wargoth** [SynthstromAudible/DelugeFirmware#756] [Ideas]
I wonder if it's possible to add compressor on per-track basis. Modern sound is impossible without a compression

**Comment by m-m-adams** [comment]
Yes - will be in the next release

**Reply by ok-reza** [reply]
speaking of, i noticed that Sidechain is labeled as COMP in the Mod Matrix. perhaps that could be changed to 'SIDE' not only for clarity but also for future compression features.
